tests/ui/traits/negative-bounds/opaque-type-unsatisfied-bound.rs

```diff
@@ -13,9 +13,17 @@ fn main() {
 }
 
 fn weird0() -> impl Sized + !Sized {}
-//~^ ERROR type mismatch resolving `impl !Sized + Sized == ()`
+//~^ ERROR the size for values of type `()` cannot be known at compilation time [E0277]
```

Comment:
that's such a scuffed error xx please add a `FIXME(negative_bounds)` to this test

Reply:
https://github.com/rust-lang/rust/pull/131701 fixes this

Reply:
gamer

Reply:
I guess let's wait with this PR until #131701 is merged then? :thinking: